server/src/main/scala/com/stratio/crossdata/server/actors/ServerActor.scala

```diff
@@ -172,8 +172,16 @@ class ServerActor(cluster: Cluster, sessionProvider: XDSessionProvider, serverAc
       sender ! ClusterStateReply(sc.cmd.requestId, cluster.state)
 
     case sc@CommandEnvelope(_: OpenSessionCommand, session) =>
-      sessionProvider.newSession(session.id)
-      sender ! ClusterStateReply(sc.cmd.requestId, cluster.state)
+      sessionProvider.newSession(session.id) match {
+        case Success(_) =>
+          logger.debug(s"new session with sessionID=${session.id} has been created")
+          sender ! OpenSessionReply(sc.cmd.requestId, isOpen = true)
+        case Failure(error) =>
+          logger.error(s"failure while creating the session with sessionID=${session.id}")
+          sender ! OpenSessionReply(sc.cmd.requestId, isOpen = false)
+      }
```

Comment:
This is silly, just personal preference concerned on making easier future changes of `OpenSessionReply`: 

```scala
val open = sessionProvider.newSession(session.id) match {
    logger.debug(s"new session with sessionID=${session.id} has been created")
    true
  case Failure(error) =>
    logger.error(s"failure while creating the session with sessionID=${session.id}")
    false
}
sender ! OpenSessionReply(sc.cmd.requestId, isOpen = open)
```